client/properties.lua

```diff
@@ -220,15 +220,14 @@ local function modifyProperty(propertyData)
     }, function(_, _, args)
         if not args then return end
         if args.action == "name" then
-            local propertyString = string.split(propertyData.name, ' ')
-            local propertyNumber = tonumber(propertyString[1])
-            local input = lib.inputDialog(Lang:t('modify_property_menu.title'), {
-                {type = 'input', label = Lang:t('modify_property_menu.name', {name = propertyData.name}), default = table.concat(propertyString, ' ', 2), required = true},
+            local propertyNumber, propertyString = string.strsplit(' ', propertyData.name, 2)
+            local input = lib.inputDialog(locale('modify_property_menu.title'), {
+                {type = 'input', label = locale('modify_property_menu.name', propertyData.name), default = propertyString, required = true},
```

Comment:
`default = propertyString` doesn't match the previous way this worked, did you purposefully not include `propertyNumber`? Was including it a bug?

Reply:
It worked the same, it would split the whole string in a table, then for the default value, concatenate it all into a string again ***starting from the second value***, so it would skip the number.

Reply:
You only modify the name of the property not the number (since that's the db id, could be useful for staff to easily identify it, i guess)